README.md

```diff
@@ -95,24 +95,27 @@ To quickly start using Passkeys in your Django project, follow these steps:
    # The name of the relying party (RP). This is sometimes shown to the user when they register a Passkey.
    OTP_WEBAUTHN_RP_NAME = "My Website Inc."
    # This is necessary to bind the Passkey to a specific domain. This should be the domain of your website.
-   OTP_WEBAUTHN_RP_ID = "your-domain.com"
+   OTP_WEBAUTHN_RP_ID = "localhost"
    # This is used to check the origin of the request and is used for security. It is similar to Django's CSRF_TRUSTED_ORIGINS setting.
    # The origins must always be a subdomain of the RP ID or the RP ID itself.
-   OTP_WEBAUTHN_ALLOWED_ORIGINS = ["https://your-domain.com", "https://subdomain.your-domain.com"]
+   OTP_WEBAUTHN_ALLOWED_ORIGINS = ["http://localhost:8000"]
 
    ```
 
 5. Add `django_otp_webauthn.backends.WebAuthnBackend` to `AUTHENTICATION_BACKENDS` in your Django settings. This step is required to make 'passwordless authentication' work.
```

Comment:
I reviewed your PR and updated my code to reflect your changes, and I believe it’s working now. However, I noticed a slight difference in how the notification is displayed on Chrome versus Safari.

That said, I don’t think it affects the functionality, as both display similar messages. For context, I’m using a Mac M2 for testing.

Reply:
The notification _content_ is not something controlled by this package – we just trigger the notification itself. You are seeing the specific implementation of Chrome and Safari.